modules/mobile/app/services/mobile/v2/appointments/proxy.rb

```diff
@@ -42,38 +63,16 @@ def merge_clinics(appointments)
           end
         end
 
-        def merge_facilities(appointments)
```

Comment:
the diff here looks off because I changed the order of when a method is defined to group them up better.